rosdistro_reviewer/submitter/github.py

```diff
@@ -16,9 +16,30 @@
+
+If you'd like to run this tool again to generate a new review for your PR, just [re-request review from GitHub Actions](https://github.com/ros-infrastructure/rosdistro-reviewer?tab=readme-ov-file#running-rosdistro-reviewer-in-github-actions).
+
+# ROS Distro Considerations
+* ROS Distribtutions are created using [REP-134 Standards Track](https://ros.org/reps/rep-0143.html) as a guide.
+* Your package name should comply to [REP-144 ROS Package Naming](https://www.ros.org/reps/rep-0144.html)
+* Your pacakge must build for all platforms and architectures on the ROS buildfarm. See [REP-2000 ROS Releases and Supported Platforms](https://www.ros.org/reps/rep-2000.html) for all supported platforms for your ROS Distro.
+* Your package must contain an [OSI approved license](https://opensource.org/licenses). Your `package.xml` file must also include that license in a machine readable format. See [REP-149 Package Manifest Format Three Specification](https://ros.org/reps/rep-0149.html#license-multiple-but-at-least-one) for additional details.
+* Your package must include a copyright at the top of every file.
+* A publicly available, open source, repository for your ROS package.
+* While not required, we recommend that you create an account for ROS Discourse and subscribe to the [appropriate release topic](https://discourse.ros.org/c/release/16).
+* If you real time chat would help you resolve an issue please join our [Discord Server] and join the `#infra-help` channel.
+
+# Package Considerations
+
+Having your package included in a ROS Distro is a badge of quality, and we recommend that package developers strive to create packages of the highest quality. We recommend package developers review the following resources before submitting their package.
+
+* [REP-2004 Package Quality Declaration](https://www.ros.org/reps/rep-2004.html)-- The ROS 2 TGC has created a quality rating system for ROS packages. These ratings should serve as a guide for package developers. We recommend package developers achieve a quality level of three or higher.
+* Documentation -- at minimum ROS packages should include an extensive [README.md file, and API level documentation using the Sphinx documentation system](https://docs.ros.org/en/rolling/How-To-Guides/Documenting-a-ROS-2-Package.html).
+* Maintainer Responsibilities -- the ROS 2 documentation includes a guide to [ROS package maintainer responsibilities](https://docs.ros.org/en/rolling/How-To-Guides/Core-maintainer-guide.html) that summarizes your responsibilities as an open source maintainer.
```

Comment:
The guidelines there I believe are for ROS 2 _core_ maintainers rather than community package maintainers. As long as there's nothing too controversial in there they should be applicable but I don't know that they're for all maintainers by design.

Reply:
We could add a little disclaimer that they're not enforced outside of the core team, but we consider it best practices.